src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -1033,6 +1033,63 @@ ov::runtime::Tensor generate(const
     return generate(std::dynamic_pointer_cast<ov::Node>(node), port, elemType, targetShape);
 }
 
+ov::runtime::Tensor generate(const
+                             std::shared_ptr<ov::op::v6::ExperimentalDetectronDetectionOutput>& node,
+                             size_t port,
+                             const ov::element::Type& elemType,
+                             const ov::Shape& targetShape) {
+    InputGenerateData inGenData(1, 0, 1, 1);
+    auto tensor = ov::test::utils::create_and_fill_tensor(elemType, targetShape, inGenData.range, inGenData.start_from, inGenData.resolution, inGenData.seed);
+
+    if (0 == port || 1 == port) {
+#define CASE(X) case X: { \
+        using T = typename ov::fundamental_type_for<X>; \
+        auto raw_ptr = tensor.data<T>(); \
+        if (port == 0) { \
+            raw_ptr[2] = static_cast<T>(10.f); \
+            raw_ptr[3] = static_cast<T>(10.f); \
+        } \
+        if (port == 1) { \
+            raw_ptr[0] = static_cast<T>(5.f); \
+        } \
```

Comment:
Am I right that it's a port of single_layer\experimental_detectron_detection_output.cpp `generateInputTensors()`? Looks like values `4.0f`, `5.0f`, `8.0f` are missed

Reply:
Yes, I did it intentionally. 
Port 0 and 1 are filled with boxes. Shapes are always (4 * n) and (4 * m).
There are always at least 4 elements as input. So I can fill them without any extra check.

For other elements to fill, I have to check for tensor size. I wanted to rid of extra ifs